### PR TITLE
[staging] fix: Fixed input height, disabled More  button & added header [123]

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

In this PR i have done changes after demo assignments
Tha changes i have done  in the PR included :-
1.Increase input URL height to 36px
2.Disabled more button in side bar
2.Added icon and workspace name in the Header tab (hard coded the workspace name) for now , also added the tooltip while hovering the workspace name

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@library/forms/select/Select.svelte [hunk 1]
```diff
@@ -51,7 +51,7 @@
   /**
    * Determines the dimensions of a Select.
    */
-  export let headerHeight = "34px";
+  export let headerHeight = "36px";
   export let maxBodyHeight = "200px";
   export let minHeaderWidth = "50px";
```
Comment (itsmdasifraza) [inline src/packages/@library/forms/select/Select.svelte:56]: revert this changes, pass value as a props

File: src/packages/@common/components/header/Header.svelte [hunk 2]
```diff
@@ -28,7 +33,26 @@
...
 
 <header class="ps-1 pe-3 d-flex align-items-center justify-content-between">
-  <div></div>
+  <div class="d-flex ms-3 justify-content-center align-items-center">
+   
+    <div>
+      <!-- <img src={SparrowLogo} alt="Logo"> -->
+      <SparrowIcon height="17px" width="17px" color="#1193F0" />
+    </div>
+    <Tooltip title={"Coming Soon"} placement={"right"}>
```
Comment (itsmdasifraza) [inline src/packages/@common/components/header/Header.svelte:42]: pass value from variables

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]
